server/src/Scripting/v8/V8PlayerImpl.cpp

```diff
@@ -1265,6 +1268,29 @@ void Player_Function_TriggerAction(const v8::FunctionCallbackInfo<v8::Value>& ar
 	}
 }
 
+// Player Function : player.triggerclient(wep, args) -> onActionClientSide
+void Player_Function_TriggerClient(const v8::FunctionCallbackInfo<v8::Value>& args)
```

Comment:
fn signature:

player.triggerclient("gui" or "weapon", "weaponName", "args0", "arg1" ... )

source: https://forums.graalonline.com/forums/showthread.php?t=134267231

Reply:
just check if arg[0] is gui or weapon, pretty sure there is a possible "npc"